templates/contribute/index.md

```diff
@@ -12,7 +12,47 @@ We also strongly encourage setting your display name on Zulip to be your real na
    - The explanation of [naming conventions](naming.html).
    - The [documentation guidelines](doc.html).
 
-Once you have code that you'd like to contribute, you should open a PR.
+## What to contribute to mathlib
+
+Small fixes (for example fixes in docstrings) and single-lemma additions in already-existing theories
+are almost always welcome as contributions to mathlib. Longer PRs which extend existing theories are also almost
```

Comment:
We may want to add somethiing here about starting a discussion on Zulip before opening large numbers of PRs (>10?), even if they individually only contain small fixes.